### PR TITLE
Equal voting window # 2

### PR DESCRIPTION
**Pull request by hongaar**
There's now an extended voting window for slow or controversial PRs, thought we might as well skip the _preferential treatment_ of timezones.

Related: #282

### CODE REVIEW

**Comment by flibustier** [comment]
St. hongaar, holy advocate of equality 😄

**Comment by PlasmaPower** [comment]
:+1: The meritocracy means we don't have to worry about sneaky changes as much, so we can remove the "after hours".

Did you mean to remove `AFTER_HOURS_VOTE_WINDOW` and related settings from `settings.py`? It looks like you didn't touch them.

**Comment by hongaar** [comment]
@PlasmaPower got left behind, fixed now, thanks!

**Review by phil-r — APPROVED** [review]

**Comment by phil-r** [comment]
@hongaar I've fixed linting issues, the other one will be fixed in #404

**Comment by hongaar** [comment]
@phil-r awesome, thanks!

btw wonder how GitHub determines who becomes the author of the squashed commit now... :)

**Review by PlasmaPower — APPROVED** [review]

**Review by andrewda — APPROVED** [review]

**Comment by phil-r** [comment]
@hongaar I think it's the author of the PR, but lets see

**Comment by PlasmaPower** [comment]
My review without a vote was intentional. I'm neutral to this change. Maybe we could do something based on common commit times (you can get that from the GitHub API).

**Comment by PlasmaPower** [comment]
@phil-r It is the author of the PR. Your review was before the last commit though.

**Comment by PlasmaPower** [comment]
There's really no way to enforce this, but I'd prefer we not vote on changes we make.

**Comment by hongaar** [comment]
@PlasmaPower what do you mean?

**Comment by PlasmaPower** [comment]
@hongaar make the voting window shorter in periods where more commits happen.

Edit: if you're talking about my last comment, then: phil-r committed to your branch, so I'd prefer he not vote on this PR after his changes.

**Comment by phil-r** [comment]
@PlasmaPower I don't know how to remove review xD

**Comment by PlasmaPower** [comment]
@phil-r You're fine there - it was for a previous commit so it doesn't count. Once #396 gets merged (38 minutes unless changes happen) you can re-review the commit and the newer review will count.

**Comment by phil-r** [comment]
https://github.com/blog/2265-dismissing-reviews-on-pull-requests Soooo confusing, can't find this in the ui

**Comment by PlasmaPower** [comment]
@phil-r that's for maintainers of the repo, not the author of the review

File: github_api/voting.py
Comment (reddraggone9) [inline github_api/voting.py:201]: This could set directly in `settings.py` (e.g. `DEFAULT_VOTE_WINDOW_SECONDS`) instead of having a function since it's so simple now. There's no need to recalculate it every time.

**Review by reddraggone9 — APPROVED** [review]

**Pull request merged**
